scripts/run_lighthouse_tests.py

```diff
@@ -188,10 +192,17 @@ def main(args: Optional[List[str]] = None) -> None:
     else:
         lighthouse_mode = LIGHTHOUSE_MODE_PERFORMANCE
         server_mode = SERVER_MODE_PROD
-
     if lighthouse_mode == LIGHTHOUSE_MODE_PERFORMANCE:
-        print('Building files in production mode.')
-        build.main(args=['--prod_env'])
+        if not parsed_args.skip_build:
+            # Also build webpack.
+            print('Building files in production mode.')
+            build.main(args=['--prod_env'])
+        else:
+            # Skip webpack build if skip_build flag is passed.
+            print('Building files in production mode skipping webpack build.')
+            build.main(args=[])
+            common.run_ng_compilation()
+            run_webpack_compilation()
     else:
```

Comment:
Add a statement explaining when the `else` statement might get executed.

Reply:
done.